introduction/java_end_of_file/Solution.java

```diff
@@ -0,0 +1,13 @@
+import java.util.Scanner;
+
+public class Solution {
+  public static void main(String[] args) {
+    Scanner input = new Scanner(System.in);
+    for (int lineNumber = 1; input.hasNext(); lineNumber++) {
+      String nextLine = "";
```

Comment:
`String.format(...)` as mentioned in the previous comment.

Specifically: 

    String.format("%d %s", lineNumber, input.nextLine());